### PR TITLE
Change from spectral displacements to spectral accelerations

### PR DESCRIPTION
**Pull request by jpa198**
This is a fix for the changes made here:
https://github.com/ucgmsim/IM_calculation/pull/70

### CODE REVIEW

**Review by claudio525 — DISMISSED** [review]

**Comment by jpa198** [comment]
I've tested this with a copy of Hossack I had lying around.
The new figures line up with those of the adjusted old figures, and are a lot closer to geom than the original calculations.

Old:
station | component | pSA_0.02 | pSA_0.05 | pSA_0.1 | pSA_0.2 | pSA_0.3 | pSA_0.4 | pSA_0.5 | pSA_0.75 | pSA_1.0 | pSA_2.0 | pSA_3.0 | pSA_4.0 | pSA_5.0 | pSA_7.5 | pSA_10.0
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
WRAV | geom | 0.06013336675273 | 0.089115745322454 | 0.125133181113855 | 0.121005303566589 | 0.144512778030697 | 0.103801783325256 | 0.080523663769336 | 0.067167579184118 | 0.028598548143978 | 0.01028351531141 | 0.004587755557469 | 0.001993342513191 | 0.001102070004063 | 0.000443169802414 | 0.000250627974779
WRAV | rotd50 | 6.2335278555554E-07 | 5.72149302246633E-06 | 2.96559252638543E-05 | 0.000149028614283 | 0.000344619064028 | 0.00042019840063 | 0.000535445191359 | 0.000950297921658 | 0.000755494339119 | 0.001070922696095 | 0.001041079503484 | 0.000849442741323 | 0.000697894240638 | 0.000575025983932 | 0.000597973586903

Old with w^2 applied:
station | component | pSA_0.02 | pSA_0.05 | pSA_0.1 | pSA_0.2 | pSA_0.3 | pSA_0.4 | pSA_0.5 | pSA_0.75 | pSA_1.0 | pSA_2.0 | pSA_3.0 | pSA_4.0 | pSA_5.0 | pSA_7.5 | pSA_10.0
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
WRAV | geom | 0.06013336675273 | 0.089115745322454 | 0.125133181113855 | 0.121005303566589 | 0.144512778030697 | 0.103801783325256 | 0.080523663769336 | 0.067167579184118 | 0.028598548143978 | 0.01028351531141 | 0.004587755557469 | 0.001993342513191 | 0.001102070004063 | 0.000443169802414 | 0.000250627974779
WRAV | rotd50 | 0.061522453957503 | 0.090350196344537 | 0.117076900201006 | 0.147085346741202 | 0.151166836934552 | 0.103679799604779 | 0.084554115474825 | 0.066695570132952 | 0.02982572101747 | 0.010569583354605 | 0.004566685710875 | 0.002095915954559 | 0.001102070411023 | 0.000403575394258 | 0.000236070509801

New:
station | component | pSA_0.02 | pSA_0.05 | pSA_0.1 | pSA_0.2 | pSA_0.3 | pSA_0.4 | pSA_0.5 | pSA_0.75 | pSA_1.0 | pSA_2.0 | pSA_3.0 | pSA_4.0 | pSA_5.0 | pSA_7.5 | pSA_10.0
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
WRAV | geom | 0.06013336675273 | 0.089115745322454 | 0.125133181113855 | 0.121005303566589 | 0.144512778030697 | 0.103801783325256 | 0.080523663769336 | 0.067167579184118 | 0.028598548143978 | 0.01028351531141 | 0.004587755557469 | 0.001993342513191 | 0.001102070004063 | 0.000443169802414 | 0.000250627974779
WRAV | rotd50 | 0.061522453957503 | 0.090350196344537 | 0.117076900201006 | 0.147085346741202 | 0.151166836934552 | 0.103679799604779 | 0.084554115474825 | 0.066695570132952 | 0.02982572101747 | 0.010569583354605 | 0.004566685710875 | 0.002095915954559 | 0.001102070411023 | 0.000403575394258 | 0.000236070509801

**Review by claudio525 — DISMISSED** [review]

File: IM_calculation/IM/im_calculation.py [hunk 1]
```diff
@@ -84,19 +84,19 @@ def check_rotd(comps_to_store: Iterable[Components]) -> bool:
 
 
 def calculate_rotd(
```
[the remark this answers is not in the document]
Comment (jpa198) [reply IM_calculation/IM/im_calculation.py:86]: It's not actually for pSAs only. I've changed the variable name to reflect this

File: IM_calculation/IM/im_calculation.py [hunk 2]
```diff
@@ -84,19 +84,19 @@ def check_rotd(comps_to_store: Iterable[Components]) -> bool:
...
-    spectral_accelerations,
+    accelerations,
     comps_to_store: List[Components],
     func=lambda x: np.max(np.abs(x), axis=1),
 ):
     """
-    Calculates rotd for given spectral accelerations
-    :param spectral_accelerations: An array with shape [periods.size, nt, 2] where nt is the number of timesteps in the original waveform
+    Calculates rotd for given accelerations
+    :param accelerations: An array with shape [periods.size, nt, 2] where nt is the number of timesteps in the original waveform
```
Comment (jasonmotha) [inline IM_calculation/IM/im_calculation.py:93]: Where periods.size is 1 for IMs without period?

Comment (jpa198) [reply IM_calculation/IM/im_calculation.py:93]: Yep, the non-pSA IMs give the acceleration array an extra dimension so it fits. 
This line could probably also be changed to [..., [0,1]] for a more robust result.

**Review by claudio525 — APPROVED** [review]

**Review by sungeunbae — APPROVED** [review]

**Comment by jpa198** [comment]
PR superseded by #108

**Pull request closed without merging**